irods/session.py

```diff
@@ -67,6 +67,52 @@ def available_permissions(self):
             self.__access = _iRODSAccess_pre_4_3_0 if self.server_version < (4,3) else iRODSAccess
         return self.__access
 
+    # TODO - write tests for new interface: sessionObject.groups
```

Comment:
Is this done?

Reply:
These have been added as well :
`test_group_admin_can_create_and_administrate_groups_issue_426`
`test_group_admin_can_create_users__issue_428`

Reply:
"and_administrate_groups" -> "and_administer_groups"

Reply:
Ah, ... right. Done.

Reply:
Ok to resolve this? Assuming that I'm going to:
   1. remove the `# TODO...` comment we discussed above. with no extra tests considered necessary, and
   2. nullify all changes I've made within the existing `user_groups` interface - with the exception of printing a deprecation message for its use.

Reply:
Yes, I think that once you have done those two things, this can be resolved.